<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionManager</class>
 <widget class="QMainWindow" name="VersionManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <!-- Os radio buttons serÃ£o adicionados aqui dinamicamente -->
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnApply">
        <property name="text">
         <string>Aplicar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUpdate">
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
